feature-libs/asm/core/connectors/asm.adapter.ts

```diff
@@ -11,4 +11,6 @@ export abstract class AsmAdapter {
   abstract customerSearch(
     options: CustomerSearchOptions
   ): Observable<CustomerSearchPage>;
+
+  abstract bindCart(cartId: string, customerId: string): Observable<void>;
```

Comment:
`Observable<void>` should probably be something else than void.  Let's look at other similar functions to see what type is used in similar cases.